src/test/resources/data/StudentProfilePageUiTest.json

```diff
@@ -37,13 +37,43 @@
       "isInstructor": false,
       "email": "SHomeUiT.benny.c@gmail.tmt",
       "institute": "TEAMMATES Test Institute 1"
+    },
+    "instructor1OfTestingSanitizationCourse": {
```

Comment:
This is commonly done if the account is just used for one course though, you can check out typicalDataBundle.json and InstructorCourseStudentDetailsPageUiTest.json.